src/microbe_stage/Compound.cs

```diff
@@ -20,6 +20,16 @@ public class Compound : IRegistryType
 
     public bool IsCloud;
 
+    /// <summary>
+    ///   Scales the retention rate of this compound as a cloud in the environment (DO NOT set above 1!)
+    /// </summary>
+    public float DecayRate = 1.0f;
```

Comment:
Why isn't this in the JSON parameters files, like `IsAgent` property? 

Also, for the maximum value, perhaps it would then be adequate to use `Math.Clamp(0.0f, 1.0f)` (syntax not guaranteed) ? Or, alternatively, use exceptions if this should really not go unnoticed.

Reply:
The "Check" method is meant to verify the data is good. So instead of having this in a comment it should be added there to make sure the game doesn't start with incorrect parameters.